### ISSUE TITLE
Local setup Docker issues

### ISSUE THREAD

**Issue by abhinaynagpal** [authzed/spicedb#182]
Hi, 

I am  interested in exploring and contributing to this project. I was following the steps to run it locally using docker, I noticed the following 

1) docker run quay.io/authzed/spicedb
>>>
A database that stores, computes, and validates application permissions

Usage:
  spicedb [command]

Examples:
	No TLS and in-memory:
		spicedb serve --grpc-preshared-key "somerandomkeyhere" --grpc-no-tls
...

this is because the Dockerfile CMD ["spicedb"] needs these args, which however cannot be supplied as part of environment flags.

**Comment by jzelinskie** [comment]
Alternatively the entrypoint can be the first argument:

```sh
docker run quay.io/authzed/spicedb spicedb serve --grpc-preshared-key "somerandomkeyhere" --grpc-no-tls
```

**Comment by jzelinskie** [comment]
This should all have been fixed as of the v1.1.0 release.
